frmts/msgn/msgndataset.cpp

```diff
@@ -61,16 +67,16 @@ class MSGNDataset final: public GDALDataset
 
     Msg_reader_core*    msg_reader_core;
     open_mode_type      m_open_mode = MODE_VISIR;
-    bool                m_bHRVDealWithSplit = false;
+    hrv_mode_type       m_bHRVDealWithSplit = NOT_HRV;
```

Comment:
The 'b' means boolean, hence changing to 'e' to mean enumeration:

```suggestion
    hrv_mode_type       m_eHRVDealWithSplit = NOT_HRV;
```